docs/migrations/migrate_renderer.md

```diff
@@ -0,0 +1,228 @@
+---
+id: migrate_renderer
+title: From legacy to Decentralize renderer
```

Comment:
How about "From Legacy to Decentralize Renderer"?

Reply:
done